openwisp_controller/config/whois/commands.py

```
@@ -0,0 +1,66 @@
+from django.core.management.base import CommandError
+from django.db.models import OuterRef, Subquery
+from swapper import load_model
+
+from openwisp_controller.config import settings as app_settings
+
+
+def clear_last_ip_command(stdout, stderr, interactive=True):
```

Ruff (0.15.0)

[warning] Unused function argument: `stderr`

(ARG001)